### ISSUE TITLE
Feature Request : Add more dimensionless units : ppm, ppb, ‱, etc...

### ISSUE THREAD

**Issue by fblgdc** [Proektsoftbg/Calcpad#376]
For now % (percent) and ‰ (permille) dimensionless units are supported.
It would be nice to also have : 
- ‱  : per 10,000 
- pcm : per 100,000 
- ppm : per million 
- ppb : per billion
(see https://en.wikipedia.org/wiki/Parts-per_notation)

**Comment by fblgdc** [comment]
The custom units does not work well with dimensionless units. See the following code :

```
'1 Ampere multiplied by 1ppm equals 1μA => works fine
1A*0.0001%|μA

'Same calculus, does not work properly
.ppm = 0.0001%
1A*1ppm
1A*1ppm|μA
```

**Comment by Proektsoftbg** [comment]
Hi!

I added these in version 7.1.0:
https://github.com/Proektsoftbg/Calcpad/releases/tag/v7.1.0.

Dimensionless units: %, ‰, ‱, pcm, ppm, ppb, ppt, ppq;

**Comment by fblgdc** [comment]
Thank you for the quick feedback!
Everything seems to work fine, except for the permyriad symbol, for which I get the following error : 
Error in "1‱" on line [12]: Invalid symbol: "‱".

I used the symbol directly from the documentation page :
![image](https://github.com/user-attachments/assets/5742bec8-a602-4f1a-b2b8-c9c23a5e21a3)

**Comment by sononicola** [comment]
I think this update added a bug with `|%`.  Surely changed the behaviur from the 7.19 since broked some my old code

![immagine](https://github.com/user-attachments/assets/9eaba60e-e6e8-4bd5-85ee-6a38d6d78bc6)

In my opinion `0.14|%` should give 14% aka 0.14

**Comment by Proektsoftbg** [comment]
@sononicola, yes you are absolutely right. Thank you! I will fix that too.

**Comment by Proektsoftbg** [comment]
Hi! Everything is now fixed in version 7.1.1:
https://github.com/Proektsoftbg/Calcpad/releases/tag/v7.1.1
